app/src/main/java/instinctools/android/utility/Common.java

```diff
@@ -0,0 +1,24 @@
+package instinctools.android.utility;
+
+import android.content.Context;
+import android.os.Environment;
+
+import java.io.File;
+
+import static android.os.Environment.isExternalStorageRemovable;
+
+/**
+ * Created by orion on 24.12.16.
+ */
+
+public class Common {
+
+    public static File getDiskCacheDir(Context context) {
```

Comment:
`android.permission.READ_EXTERNAL_STORAGE` and `android.permission.WRITE_EXTERNAL_STORAGE` requires runtime permission request https://developer.android.com/training/permissions/requesting.html.